### PR TITLE
blog/: add 3mdeb roadmap for August

### PR DESCRIPTION
**Pull request by artur-rs**
Signed-off-by: Artur Raglis <artur.raglis@3mdeb.com>

### CODE REVIEW

**Comment by pietrushnic** [comment]
@artur-rs IMO we should have more names of technologies, Linux kernel, U-Boot, devices tree, coreboot, libpayload, Bareflank, hypervisors etc.

**Comment by lwcislo** [comment]
> @artur-rs IMO we should have more names of technologies, Linux kernel, U-Boot, devices tree, coreboot, libpayload, Bareflank, hypervisors etc.

I agree.

**Comment by artur-rs** [comment]
@pietrushnic @lwcislo is it better now?

**Comment by pietrushnic** [comment]
@artur-rs you treated my suggestion very directly and BTW I didn't submit my previous review :man_facepalming: , for example, there is no word about the effort of preparing to conferences but we mention Bareflank.
@lwcislo could you use your magic skills to pimp that report? Max 20-30min

File: blog/content/post/2019-08-16-3mdeb-roadmap.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: '3mdeb roadmap for August 2019'
+abstract: 'If you are interested in upcoming 3mdeb projects, this post will
```
[the remark this answers is not in the document]
Comment (lwcislo) [reply blog/content/post/2019-08-16-3mdeb-roadmap.md]: If you are interested in technologies we work this month, this post will explain in a few sentences the work plan for August 2019. Find out what technologies will be used by the 3mdeb staff -> Thrilled about what we are doing to change the world right now? Interested in the fancy stuff we make? Check out our plan for August 2019, find out more about our recent projects and stay tuned with the most sophisticated tech-team in the neighborhood.

**Pull request merged**